examples/chat2-reliable/chat.go

```diff
@@ -26,9 +28,14 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
-// Chat represents a subscription to a single PubSub topic. Messages
-// can be published to the topic with Chat.Publish, and received
-// messages are pushed to the Messages channel.
+const (
+	maxMessageHistory   = 100
```

Comment:
Perhaps we also want to make the number of IDs kept in the causal history configurable?